src/inventory/gilded_rose.js

```diff
@@ -22,49 +22,77 @@ const items = [
 updateQuality(items);
 */
 export function updateQuality(items) {
+    //combine these into quality changer--takes in a + or - number and you can get BOTH increase or decrease in quality
+    qualityUpdater = (item, change_Amount) =>{
```

Comment:
Just a heads up that this function is still scoped inside of `updateQuality`. Yesterday, we chatted about bringing this function outside of `updateQuality` to eliminate unintended sideeffects.

Reply:
@jordanleven I just chatted with @tisidro and after looking at this comment I may have l misunderstood what you were telling her the problem was. Is the problem just how this function is scoped or is it side effects in general? @tisidro  If the problem is just how the function is scoped you can disregard what I said about side effects at the end of our pairing. What you showed me fixed that problem.